server/src/models/finances/constants.js

```diff
@@ -1,18 +1,18 @@
 import _ from "lodash";
 import {
   public_account_years,
-  planning_years,
+  correct_planning_years,
```

Comment:
Nothing in this file is used anyways. Is this safe to delete?

Reply:
I think that's more because the finance section of the API is old/unfinished, since we haven't decided to move to it yet. In theory this stuff would be used eventually, better to keep it around/up to date maybe.